config-tests/test_remote_settings.py

```diff
@@ -75,3 +75,20 @@ def test_create_collection_in_main_bucket(conf, env):
         id=collection,
         data={"status": "to-sign"}
     )
+
+    # Now clean up all the records we created
+    dc_response = client.delete_collection(id=collection, bucket='main-workspace')
+    assert dc_response['deleted'] is True
+
+    dc_response = client.delete_collection(id=collection, bucket='main-preview')
+    assert dc_response['deleted'] is True
+
+    dc_response = client.delete_collection(id=collection, bucket='main')
+    assert dc_response['deleted'] is True
```

Comment:
nitpick: could have used a loop `for bid in ('main-workspace', 'main-preview', 'main'):`

Reply:
Not using a loop is a holdover from my PHP days where it was considered a "test smell" to have loops and conditional statements inside your tests. I will update it